### DISCUSSION TITLE
🎉 Welcome to QuizCast! 🚀

### DISCUSSION THREAD

**Discussion by Tharindu209** [QuizCast/frontend#71] [General]
We’re thrilled to officially announce the launch of QuizCast, a dynamic and interactive platform for real-time quiz challenges! Whether you’re a trivia enthusiast, an educator, or simply love friendly competitions, QuizCast is here to make quizzing fun and engaging.

QuizCast is an open-source platform that lets you:
- Host quizzes effortlessly.
- Participate in real-time challenges.
- Experience seamless interactions powered by cutting-edge technology like Supabase and FastAPI.

Our mission is to redefine online quizzes with a platform that’s:
✅ Intuitive
✅ Real-time
✅ Community-driven

🚀 Key Features

- Supabase-Powered Real-Time Updates: Enjoy instant leaderboard updates and live submissions.
- Secure Authentication: Powered by JWT tokens for safe and seamless access.
- Customizable Quizzes: Create unique quizzes and share quiz keys with participants.
- Responsive UI: Built using Next.js, Tailwind CSS, and Flowbite.
- Open Source: Entirely open for contributions under the MIT License.

How You Can Contribute:

- Start by forking a repo, working on an issue, or suggesting features.
- Share your expertise and join the conversation in GitHub Discussions.

💡 What’s Next?
This is just the beginning! We're excited to bring new features, improve real-time interactions, and grow the QuizCast community.

🖇️ Connect With Us
Have questions or ideas? We’d love to hear from you!

- Open a discussion in our repos.
- Reach out to us directly.

Let’s build something amazing together. Join us on this journey to redefine quizzes, one question at a time!

🎊 Start exploring now at [QuizCast](https://quizcast-teamvertex.vercel.app/)

**Comment by ghost** [comment]
if you could give sql editor codes, it would be more beneficial. thanks a lot.
